Tests/scripts/convert_test_result_to_jira_issues.py

```diff
@@ -0,0 +1,183 @@
+                    jira_server.transition_issue(searched_issue, unresolved_transition['id'])
+                    jira_issue = searched_issue
+                else:
+                    logging.error(f"Failed to find the '{JIRA_ISSUE_UNRESOLVED_TRANSITION_NAME}' "
+                                  f"transition for issue {searched_issue.key}")
+                    jira_issue = None
+                    use_existing_issue = False
+                    link_to_issue = searched_issue
+
+            else:
+                link_to_issue = searched_issue
+        else:
+            jira_issue = searched_issue
+    return jira_issue, link_to_issue, use_existing_issue
+
+
+def generate_description(build_id: str, properties: dict[str, str], test_suite: TestSuite) -> str:
+    build_url = f"{GITLAB_SERVER_URL}/{GITLAB_PROJECT_ID}/-/pipelines/{build_id}" if build_id else ""
+    build_id_hash = f" #{build_id}" if build_id else ""
+    build_markdown_link = f"[Nightly{build_id_hash}|{build_url}]" if build_id else f"Nightly{build_id_hash}"
+    table = tabulate(tabular_data=[
+        ["Total", test_suite.tests],
+        ["Failed", test_suite.failures],
+        ["Errors", test_suite.errors],
+        ["Skipped", test_suite.skipped],
+        ["Successful", test_suite.tests - test_suite.failures - test_suite.errors - test_suite.skipped],
+        ["Duration", f"{test_suite.time}s"]
+    ], tablefmt="jira", headers=["Tests", "Result"])
+    description = f"""
+        *{properties['pack_id']}* - *{properties['file_name']}* failed in {build_markdown_link}
+
+        {table}
+        """
+    return description
+
+
+def main():
+    try:
+        install_logging('convert_test_result_to_jira_issues.log', logger=logging)
+        now = datetime.now(tz=timezone.utc)
+        options = options_handler()
+        logging.info(f'JUnit path: {options.junit_path}')
+        logging.info(f'Gitlab server url: {options.url}')
+        logging.info(f'Gitlab project id: {options.gitlab_project_id}')
+        logging.info(f'Jira server url: {JIRA_SERVER_URL}')
+        logging.info(f'Jira verify SSL: {JIRA_VERIFY_SSL}')
+        logging.info(f'Jira project id: {JIRA_PROJECT_ID}')
+        logging.info(f'Jira issue type: {JIRA_ISSUE_TYPE}')
+        logging.info(f'Jira component: {JIRA_COMPONENT}')
+        logging.info(f'Jira labels: {JIRA_LABELS}')
+        logging.info(f'Jira issue unresolved transition name: {JIRA_ISSUE_UNRESOLVED_TRANSITION_NAME}')
+        logging.info(f'Max days to reopen: {options.max_days_to_reopen}')
+
+        jira_server = JIRA(JIRA_SERVER_URL, token_auth=JIRA_API_KEY, options={'verify': JIRA_VERIFY_SSL})
+
+        xml = JUnitXml.fromfile(options.junit_path)
+        for test_suite in xml.iterchildren(TestSuite):
+            if test_suite.failures or test_suite.errors:
+                create_jira_issue(jira_server, test_suite, options, now)
+            else:
+                logging.info(f"Skipped creating Jira issue for {test_suite.name}")
```

Comment:
Maybe this should be debug level?
```suggestion
                logging.info(f"Skipped creating Jira issue for successful test {test_suite.name}")
```